core/src/main/java/brooklyn/entity/basic/Attributes.java

```diff
@@ -114,6 +115,15 @@
         "service.problems", 
         "A map of namespaced indicators of problems with a service");
 
+    /**
+     * @since 0.8.0
+     */
+    @SuppressWarnings("serial")
```

Comment:
Oh, this is acceptable? I got into the habit of just defining my `TypeToken`s with the version UUID inline.